.github/workflows/main.yml

```diff
@@ -17,6 +17,13 @@ jobs:
     name: Build
     uses: ./.github/workflows/build.yml
 
+  windows-build:
+    name: Build on Windows
+    uses: ./.github/workflows/windows-build.yml
+    with:
+      name: release
+      options: -DCMAKE_BUILD_TYPE=Release -DENABLE_FOLLY=off -G "Visual Studio 17 2022"
```

Comment:
Does folly not work on windows?

Reply:
I suppose it works but there is no conda-forge build on windows
https://github.com/conda-forge/folly-feedstock/blob/main/recipe/meta.yaml#L19

Reply:
We decided to skip it for now, which would use std lib atomics on Windows. The Arena allocator is a bigger issue we need to solve.

Reply:
Someone is trying to write windows build so maybe eventually they will add one
https://github.com/conda-forge/folly-feedstock/pull/98